force-app/main/default/classes/DummyJSONCallout.cls

```diff
@@ -33,19 +32,54 @@ public with sharing class DummyJSONCallout {
 	 * 
      * @param dummyUserId The ID of the user in the external system to fetch data for.
      */
+	
+	@future(callout=true)
 
 	public static void getDummyJSONUserFromId(String dummyUserId) {
-		// Create HTTP request to send.
 
+		// Create HTTP request to send.
+		Http http = new Http ();
+		HttpRequest req = new HttpRequest();
 		// Set the endpoint URL. Use direct URL or for best practices use Named Credential.
 
-		// Set the HTTP method to GET.
+		req.setEndpoint('https://dummyjson.com/user/' + dummyUserId);
```

Comment:
Best practice is to use Named Credentials. Be sure to know how to do this for the capstone.